File: java/src/main/java/org/extism/sdk/LogLevel.java
package org.extism.sdk;

public enum LogLevel {
    INFO("info"),
    DEBUG("debug"),
    WARN("warn"),
    TRACE("trace");

    private String level;

    LogLevel(String level) {
        this.level = level;
    }

    public String getLevel() {
        return level;
    }
}

File: dotnet/src/Extism.Sdk/LogLevel.cs
namespace Extism.Sdk.Native;

/// <summary>
/// Extism Log Levels
/// </summary>
public enum LogLevel
{
    /// <summary>
    /// Designates very serious errors.
    /// </summary>
    Error,

    /// <summary>
    /// Designates hazardous situations.
    /// </summary>
    Warning,

    /// <summary>
    /// Designates useful information.
    /// </summary>
    Info,

    /// <summary>
    /// Designates lower priority information.
    /// </summary>
    Debug,

    /// <summary>
    /// Designates very low priority, often extremely verbose, information.
    /// </summary>
    Trace
}
